examples/StateForm-list.tsx

```diff
@@ -23,24 +41,24 @@ const Demo = () => {
         style={{ border: '1px solid red', padding: 15 }}
       >
         <List name="users">
-          {(fields, { add, remove }) => {
+          {(fields, { add, remove,move }) => {
             console.log('Demo Fields:', fields);
             return (
               <div>
                 <h4>List of `users`</h4>
                 {fields.map((field, index) => (
-                  <LabelField {...field} rules={[{ required: true }]}>
+                  <DraggableField move={move} index={index} id={field.key} {...field} rules={[{ required: true }]}>
                     {control => (
-                      <div style={{ position: 'relative' }}>
-                        <Input {...control} />
-                        <a style={{ position: 'absolute', top: 12, right: -300 }} onClick={() => {
-                          remove(index);
-                        }}>
-                          Remove
-                        </a>
-                      </div>
+                        <div style={{ position: 'relative'}}>
```

Comment:
Strange, why `pre-commit` of prettier hook not work...

Reply:
not idea